app/views/content/life-as-a-teacher/age-groups-and-specialisms/special-educational-needs.md

```diff
@@ -17,32 +17,82 @@ keywords:
+
+* social, emotional and mental health 
+
+* sensory and physical needs 
 
-No matter what settings you train and work in – whether it's mainstream schools, special schools, or both – you’ll work with pupils with complex needs.
+During your initial teacher training, you may have the opportunity to have one of your school placements in a special school. This can enhance your understanding of how to plan and teach more effectively for all children.  
 
-$q-jane-wilkinson$
+### Becoming a special educational needs co-ordinator
 
-You can [find a teacher training course](https://find-teacher-training-courses.service.gov.uk/) and filter by 'courses with a SEND specialism' if you have a particular interest in the area.
+A special educational needs coordinator (SENCO) assesses, plans and monitors the progress of pupils with special educational needs and disabilities in mainstream schools. SENCOs also support teachers and teaching assistants in a range of ways including providing training, observing lessons and developing learning plans for pupils. 
 
-However, it’s not essential to do a course specialising in SEND.
+As a SENCO you will get to know the specific needs of pupils who need extra support, so you can help them to feel comfortable and confident before teaching starts. 
 
-As you train and teach you'll get the skills you need to work with disabled pupils and pupils with special educational needs.
+To become a SENCO, you’ll need to gain [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts) through initial teacher training and then complete a mandatory [SENCO national professional qualification (SENCO NPQ)](https://www.gov.uk/guidance/special-educational-needs-co-ordinators-national-professional-qualification). You do not need to complete the NPQ before taking a SENCO role, you have 3 years to complete the qualification. 
 
-This will enable you to work in a range of settings throughout your career.
+As well as focusing on supporting pupils with SEND, many SENCOs will also teach in their main subject area. 
 
-## Teach pupils with sensory impairments
+### Mandatory qualification in sensory impairment
 
-You need [specific qualifications to teach a class of pupils with hearing impairments, vision impairments or multi-sensory impairments](https://www.gov.uk/guidance/mandatory-qualifications-specialist-teachers).
+To teach pupils with hearing impairments, vision impairments or multi-sensory impairments you’ll need a [mandatory qualification in sensory impairment (MQSI)](https://www.gov.uk/guidance/mandatory-qualifications-specialist-teachers) in addition to QTS.  
 
-## Become a special educational needs coordinator
+An MQSI will give you the skills needed to recognise pupils’ barriers to learning and help them overcome these. 
 
-A special educational needs coordinator (SENCO) assesses, plans and monitors the progress of disabled pupils and pupils with special educational needs.
+## Training for teaching pupils with SEND
```

Comment:
It feels a bit odd to have this at the bottom of the page, having spoken about training earlier? But I appreciate that we want people to read the information first!